static/app/components/workflowEngine/form/inlineNumberField.tsx

```diff
@@ -0,0 +1,18 @@
+import type {ComponentProps} from 'react';
+import styled from '@emotion/styled';
+
+import NumberField from 'sentry/components/forms/fields/numberField';
+
+type NumberFieldProps = ComponentProps<typeof NumberField>;
+
+export default function InlineNumberField(props: NumberFieldProps) {
+  return <StyledNumberField style={{height: '28px', minHeight: '28px'}} {...props} />;
+}
+
+const StyledNumberField = styled(NumberField)`
+  padding: 0;
+  width: 180px;
```

Comment:
if we have to hardcode a width -- could we do it on the container and then rely on the `display` property to grow the input field or not?

Reply:
I would normally agree but in this case the designs are using the same width for every field.